src/ioc/index.ts

```diff
@@ -1,12 +1,21 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
 
 export const createIoCContainer = () =>  {
   const ioc = new IoCContainer();
-  // you can register some resources right now below...
+
+  ioc.registerClass('logger', Logger);
+  ioc.registerClass('users', Users);
+  ioc.registerClass('http', HTTP)
+  ioc.register('config', {
+    "path": "/api",
+    "resources": {
+      "users": "/users"
+    }
+  });
```

Comment:
Why do you hardcode config ?
Hint: you can register config in `src/index.ts` file, actually anywhere